### PR TITLE
New release: v2.1.0

### PR DESCRIPTION
**Pull request by alcuadrado**
@holgerd77, I missed your comment on preparing a new release in #163 when creating #166, sorry about that.

I'm opening this PR with for the new release, based on #166. It will have to be rebased/updated once that PR is merged.

### CODE REVIEW

**Comment by holgerd77** [comment]
Thanks, have merged #166, this can be updated.

**Comment by alcuadrado** [comment]
Updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.1.0] - 2018-06-26
+
+- Upgraded [ethereumjs-common](https://github.com/ethereumjs/ethereumjs-common) to `^1.3.0`
+- Added more documentation and examples on how to create transactions for public testnets and
+  custom networks
```
Comment (holgerd77) [inline CHANGELOG.md]: Hmm, I think this needs a more explicit changelog entry to allow people to grasp what is going on and what problems have been solved. Can you write up 2-4 sentences up to grab the essence from https://github.com/ethereumjs/ethereumjs-common/issues/54 (but relating to what it means for this library)?

Comment (alcuadrado) [reply CHANGELOG.md]: Makes sense, will do it tomorrow.

**Comment by alcuadrado** [comment]
Oops, that was accidental. I agree with not committing `package-lock`s for libraries.

**Pull request merged**